boards/arm64/xenvm/xenvm_gicv3_defconfig

```diff
@@ -0,0 +1,25 @@
+CONFIG_SOC_XENVM=y
+CONFIG_BOARD_XENVM=y
+
+# Use GICv3
+CONFIG_XENVM_USE_GIC_V3=y
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+CONFIG_XIP=n
```

Comment:
not needed

Reply:
this is just copied from xenvm_defconfig. The only difference between xenvm_defconfig and xenvm_gicv3_defconfig is that the latter enables GICv3 by setting CONFIG_XENVM_USE_GIC_V3

Reply:
still not needed. You can remove it from the other defconfig also.

Reply:
I can remove CONFIG_XIP=n because by default it is set to =n. However CONFIG_SERIAL is required by CONFIG_UART_CONSOLE and CONFIG_UART_XEN_HVC. So it needs to be set to =y

Reply:
sure, I was only referring to `CONFIG_XIP`